Automattic/newspack-multibranded-site
# chore: update Newspack Components

Bumps Newspack Components to [v3.1.0](https://www.npmjs.com/package/newspack-components/v/3.1.0). This should complete the fix from #58.

### Testing

1. Check out this branch
2. Run `npm ci --legacy-peer-deps` to update Node packages
3. Confirm that the radio buttons in the Multibranded Site wizard appear correctly in both WP 6.6 and WP 6.7, as described in #58
